plugins/modules/net_tools/gandi_livedns.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019 Gregory Thiemonge <gregory.thiemonge@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: gandi_livedns
+author:
+- Gregory Thiemonge (@gthiemonge)
+requirements:
+- python >= 2.6
+version_added: "1.4.0"
+short_description: Manage Gandi LiveDNS records
+description:
+- "Manages dns records via the Gandi LiveDNS API, see the docs: U(https://doc.livedns.gandi.net/)"
+options:
+  api_key:
+    description:
+    - Account API token.
+    type: str
+    required: true
+  record:
+    description:
+    - Record to add.
+    - Default is C(@).
+    type: str
+    default: '@'
+    aliases: [ name ]
+  state:
+    description:
+    - Whether the record(s) should exist or not.
+    type: str
+    choices: [ absent, present ]
+    default: present
+  ttl:
+    description:
+    - The TTL to give the new record.
+    - Default is C(10800) when C(state=present).
+    type: int
+  type:
+    description:
+      - The type of DNS record to create.
+    type: str
+    required: true
+  values:
+    description:
+    - The record values.
+    - Required for C(state=present).
+    type: list
+    elements: str
+    aliases: [ content ]
```

Comment:
```suggestion
    aliases: [ content ]
```
I noticed several aliases. It's not recommended by the doc.
Better to use one the best option now. Usually aliases is ok to add when people realize that there is a much better option or the wrong / confusing one was chosen originally or typos, etc.
Adding aliases generally makes module's API more confusing.

Reply:
Ack, for clarity, I removed the aliases

Reply:
cool thanks!